### ISSUE TITLE
run train.py have some problems

### ISSUE THREAD

**Issue by ghost** [amdegroot/ssd.pytorch#519]
I started the trainning in my pc with GPU,and platform is win10,and I fixed the batch_size = 1, and loss_c = loss_c.reshape(1, 8732) in multibox_loss.py.
![image](https://user-images.githubusercontent.com/50620362/96584941-83b54980-1311-11eb-8513-a161782d18bc.png)
in follow picture, the loss was nan,what should i do

**Comment by bspivey** [comment]
I'm having the same issue
